### DISCUSSION TITLE
I'm not able to install graph cli , pls assit me

### DISCUSSION THREAD

**Discussion by soumalya340** [graphprotocol/graph-client#423] [Q&A]
Hey, I'm trying to install "npm install -g @graphprotocol/graph-cli"  or "yarn global add @graphprotocol/graph-cli" in my Windows Ubuntu WSL but showing errors, even in Windows 11 cmd same result, can anyone help me, I need my project. Sometimes it gets downloaded but shows "graph" command not found

**Comment by ardatan** [comment]
This is the repo of Graph Client. Please create an issue with more details in the related repo.
